src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlAttributeOverrides.cs

```diff
@@ -10,16 +10,15 @@ namespace System.Xml.Serialization
     using System.Xml.Schema;
     using System;
     using System.ComponentModel;
-    // this[key] api throws KeyNotFoundException
-    using Hashtable = System.Collections.InternalHashtable;
+    using System.Collections.Generic;
 
     /// <include file='doc\XmlAttributeOverrides.uex' path='docs/doc[@for="XmlAttributeOverrides"]/*' />
     /// <devdoc>
     ///    <para>[To be supplied.]</para>
     /// </devdoc>
     public class XmlAttributeOverrides
     {
-        private Hashtable _types = new Hashtable();
+        private readonly Dictionary<Type, Dictionary<string, XmlAttributes>> _types = new Dictionary<Type, Dictionary<string, XmlAttributes>>();
```

Comment:
@khdang Maybe in a future PR, this could be converted to a `Dictionary<Member, XmlAttributes>` where `Member` is a struct containing a `Type` and a `string`.